Dockerfile

```diff
@@ -1,57 +1,5 @@
-FROM ubuntu:14.04
+FROM ubuntu:16.04
 MAINTAINER Tulio Ruiz <tulio@vauxoo.com>
 
-RUN locale-gen fr_FR \
-    && locale-gen en_US.UTF-8 \
-    && dpkg-reconfigure locales \
-    && update-locale LANG=en_US.UTF-8 \
-    && update-locale LC_ALL=en_US.UTF-8 \
-    && ln -s /usr/share/i18n/SUPPORTED /var/lib/locales/supported.d/all \
-    && locale-gen \
-    && echo 'LANG="en_US.UTF-8"' > /etc/default/locale
-
-ENV PYTHONIOENCODING="utf-8" \
-    LANG="C.UTF-8" \
```

Comment:
@LuisAlejandro We had some issues with the locales, Id like to test that with the new changes, but I can't recall what was about. @moylop260 do you remember anything about this?